src/Analyzers/CSharp/Tests/UseConditionalExpression/UseConditionalExpressionForAssignmentTests.cs

```diff
@@ -891,12 +891,12 @@ class C
 {
     void M()
     {
-        object o = true ? throw new System.Exception() : ""b"";
+        var o = true ? throw new System.Exception() : (object)""b"";
```

Comment:
This behavior change is restoring the original behavior of the test; it appears it was accidentally broken and the test baselines incorrectly updated. See the commit message for details.

Reply:
wfm.